config.py

```diff
@@ -1,19 +1,24 @@
 import os
+
 basedir = os.path.abspath(os.path.dirname(__file__))
 
-class Config(object):
+
+class Config:
     DEBUG = False
     TESTING = False
     CSRF_ENABLED = True
-    SECRET_KEY = 'this-really-needs-to-be-changed'
-    SQLALCHEMY_DATABASE_URI = 'postgresql://postgres:0000@localhost/album_cover_db'
+    SECRET_KEY = "this-really-needs-to-be-changed"
```

Comment:
could you add a ticket to sprint to look into managing secrets in a secure way.
one option:
https://firebase.google.com/docs/remote-config